CONSTITUTION.md

```diff
@@ -152,7 +153,7 @@ The Trustees shall open and maintain a bank account in the name of and on behalf
 
 ### 6.4. Withdrawals and Account Signatories
 
-Monies shall not be drawn except by cheque or orders signed by two trustees or by the Secretary for the time being of the Union who may act as sole signatory on all accounts.
+Monies shall not be drawn except by cheque or orders signed by two trustees, or by one trustee and the ActivateUTS Clubs Manager for the time being who will have been given a written explanation, or by the ActivateUTS CEO for the time being who may act as sole signatory on all accounts.
```

Comment:
1. Who provides the written explanation to whom?
2. A written explanation for what purpose?
3. What circumstances would arise that would necessitate involving the Clubs Manager?

Reply:
> Who provides the written explanation to whom?

Not currently explicit - it will be from the Executive Committee.

> A written explanation for what purpose?

So that the Clubs Manager can use their discretion as to whether it is appropriate for them to act in place of another Executive.

> What circumstances would arise that would necessitate involving the Clubs Manager?

According to ActivateUTS policy, there must be one "Primary Signatory" for every transaction (basically to ensure a UTS student is accountable, as President + Treasurer must be students at election according to Activate).
So if either are absent, we would need the Clubs Manager to co-sign to approve the transaction otherwise it would be blocked.

A previous alternative to this amendment mentioned the two trustees "involving at least the President or Treasurer".
I think I'll update the amendment accordingly given my explanation here.

Thanks @tomchristmas for making sure we properly consider the deeper consequences / reasons for these amendments, and make it explicit too.